pod/perldata.pod

```diff
@@ -164,9 +159,9 @@ Put together, a grammar to match a basic identifier becomes
           )
       )
       (?<normal_identifier>
-          (?: :: )* '?
+          (?: :: )*
            (?&basic_identifier)
-           (?: (?= (?: :: )+ '? | (?: :: )* ' ) (?&normal_identifier) )?
+           (?: (?= (?: :: )+ | (?: :: )* ) (?&normal_identifier) )?
```

Comment:
This is wrong. Since `'` is no longer accepted, the entire right branch of the original look-ahead (`(?: :: )* '`) should be removed:
```
    … (?= (?: :: )+ )
```

But the original code didn't make much sense to begin with. "Followed by 1 or more X" is equivalent to "followed by X"; we don't need to loop here. So the original regex should have been
```
    … (?: (?= :: | ' ) (?&normal_identifier) )?
```
and after removal of `'`:
```
    … (?: (?= :: ) (?&normal_identifier) )?
```

Reply:
I've fixed perldata as suggested.